src/release_on_push_action/core.clj

```diff
@@ -91,7 +91,7 @@
 ;; Note: at 500, I suspect this list will be unproductive to view
```

Comment:
Can you also remove this line, otherwise the comment about 500 won't make sense.